src/main/java/com/shaft/gui/browser/BrowserFactory.java

```diff
@@ -317,16 +321,19 @@ private static void setLoggingPrefrences() {
         logPrefs.enable(LogType.DRIVER, Level.ALL);
     }
 
-    private static void setDriverOptions(String browserName, MutableCapabilities customBrowserOptions) {
+    private static void setDriverOptions(String browserName, MutableCapabilities customBrowserOptions, Boolean appendToDefault) {
```

Comment:
I think this boolean variable is redundant, I'll share my detailed feedback offline